app/assets/stylesheets/_globals.scss

```diff
@@ -4,3 +4,8 @@ li code {
   padding: 2px 8px;
   word-break: break-word;
 }
+
+// Use NHSUK colour for <mark>
+mark {
+  background-color: $color_nhsuk-pale-yellow;
+}
```

Comment:
Has this slipped in from somewhere? I think it was moved to `_overrides.scss`

Reply:
I moved it to globals because it's closer to a global, but removed the change now.